database/influxdb/influxdb_privs.py

```diff
@@ -0,0 +1,220 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+---
+module: influxdb_privs
+short_description: Manage InfluxDB user privileges.
+description:
+    - Manage InfluxDB user privileges.
+version_added: 2.2
+requirements:
+    - "python >= 2.6"
+    - "influxdb >= 0.9"
```

Comment:
since this module requires a change into influxdb python module, i'd suggest to put version of influxdb-python module here when influxdb PR gets merged